README.md

```diff
@@ -32,7 +32,7 @@ To use the action, add it to your GitHub Actions workflow file:
 
 ## Inputs
 
-- original upload-artifact inputs can be found [here](https://github.com/actions/upload-artifact?tab=readme-ov-file#inputs)
+- see [original upload-artifact inputs](https://github.com/actions/upload-artifact?tab=readme-ov-file#inputs)
```

Comment:
##### Do not use `(click) here` links

For more information, see:
* https://www.w3.org/QA/Tips/noClickHere
* https://webaim.org/techniques/hypertext/link_text
* https://granicus.com/blog/why-click-here-links-are-bad/
* https://heyoka.medium.com/dont-use-click-here-f32f445d1021